build.md

```diff
@@ -8,13 +8,13 @@ You can build the Spacemesh Full Node and CLI wallet directly from source code,
 
 The Spacemesh App which includes the full p2p node and a wallet is supported on any Linux distro that supports [Electron Apps](https://electronjs.org/docs/tutorial/support).
 
-If you are running a Linux that doesn't support Electron than you can build and run a full p2p node and a CLI wallet.
+If you are running a Linux distribution that doesn't support Electron then you can build and run a full p2p node and a CLI wallet.
 
 --
 
 Follow the steps below to build the Spacemesh full node and CLI wallet directly from source code.
 
-Alternatively, you can also download a pre-build Linux full p2p node and CLI Wallet pre-build docker image and run them using Docker. Follow the steps in [this guide](docker.md).
+Alternatively, you can also download a pre-build Linux full p2p node and CLI Wallet pre-build docker image and run them using Docker. Follow the steps in [this guide](docker.md). 
```

Comment:
I cannot follow this sentence

Reply:
maybe because of the typo. it needs to say `prebuilt`.

Reply:
Edited, but I still think that we need to rephrased this sentence.

Reply:
Please go ahead and provide a rephrased sentence that is clearer.